compass/landice/tests/thwaites/__init__.py

```diff
@@ -3,7 +3,6 @@
 from compass.landice.tests.thwaites.restart_test import RestartTest
 from compass.landice.tests.thwaites.high_res_mesh import HighResMesh
 
-
```

Comment:
This extra newline is standard in PEP8.  Please put it back when you get a chance.

Reply:
I've added it back in.